pgql-lang/pom.xml

```diff
@@ -30,14 +30,6 @@
           <groupId>org.graalvm</groupId>
           <artifactId>graal-sdk</artifactId>
         </exclusion>
-        <exclusion>
-          <groupId>io.reactivex.rxjava3</groupId>
-          <artifactId>rxjava</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.reactivestreams</groupId>
-          <artifactId>reactive-streams</artifactId>
-        </exclusion>
```

Comment:
@ana-estrada : FYI I had to bring these two dependencies back again when upgrading from spoofax 2.5.16 to 2.5.18.

Somehow they were not needed for 2.5.16 but they are needed for 2.5.18.

Reply:
latest PGQL version is `2.0.1.6` now